test/break_test.exs

```diff
@@ -41,16 +42,18 @@ defmodule VenomousTest.BreakTest do
       slay_python_worker(%SnakeWorker{pid: pid, pypid: pypid, os_pid: os_pid}, :brutal)
     end)
 
-    args = SnakeArgs.from_params(:time, :sleep, [10])
+    args = SnakeArgs.from_params(:time, :sleep, [0.5])
 
     me = self()
 
     Enum.map(0..100, fn _ ->
       {:ok, pid} =
         Task.start(fn ->
-          python!(args)
+          capture_log(fn ->
```

Comment:
I can't quite understand what you mean. Taking python!() call out of capture_log will just bring back the :exit errors, this will also suppress any python!() function related errors but this shouldn't be a problem as we test this function itself in other tests.

Reply:
> I can't quite understand what you mean. Taking python!() call out of capture_log

I mean something like
```elixir
assert capture_log(fn -> ...) =~ "terminating" # or any other content error message worth validating
```

Reply:
Well, that's a hard one since the Task we start is exited we cannot really do anything with the captured log. I wouldn't bother with it especially that all that's happening on captured exit is the slay_python_worker being called and exit(reason) is ran again. Since we later assert if processes got killed correctly the assertion of log isn't really required. But if you really want that assertion I guess you could play with Process.monitor and :DOWN signal to check if reason is the same as is declared in  `Process.send_after(pid, {:EXIT, :left}, 200)`. Im merging the PR for now.